docker/pvs-studio.Dockerfile

```
@@ -19,3 +40,20 @@ RUN apt-get update \
     && apt-get update \
     && apt-get install -y --no-install-recommends pvs-studio pvs-studio-dotnet \
     && rm -rf /var/lib/apt/lists/*
+
+# Verify PVS-Studio installation.
+RUN pvs-studio-analyzer --version
+
+# Install .NET SDK via Microsoft's install script, AFTER the pvs-studio apt
+# packages, so this SDK (10.0.400) is the one on PATH and under DOTNET_ROOT.
+# --channel 10.0.4xx selects the latest patch in the 10.0.4xx feature band
+# (e.g. 10.0.400), matching global.json's version 10.0.400 + rollForward latestFeature.
+RUN curl -SL https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh \
```

Hadolint (2.15.1)

[warning] Either use Wget or Curl but not both

(DL4001)